QuickOSM/definitions/osm.py

```diff
@@ -23,6 +23,22 @@ class QueryType(Enum):
     AroundArea = 'AroundArea'
 
 
+@unique
+class QueryOverpassLanguage(Enum):
+    """Language of query that QuickOSM can generate."""
+    Xml = 'xml'
+    Oql = 'mapql'
+
+
+@unique
+class QueryLanguage(Enum):
+    """Language of query that QuickOSM can generate."""
+    Xml = False
+    Oql = True
+    XML = "xml"
+    OQL = "oql"
+
+
```

Comment:
```suggestion
@unique
class QueryLanguage(Enum):
    """Language of query that QuickOSM can generate."""
    Xml = "xml"
    Oql = "oql"


```

The purpose of an enum is to have a list of available languages. Not duplicated entries.
A single enum is enough, not two.

Side note, if you want to have extra attributes to an Enum, use "NamedTuple" at the same time.
https://docs.python.org/3/library/collections.html#collections.namedtuple
Example in PgMetadata plugin to store the label and the extension for a given output format :  
https://github.com/3liz/qgis-pgmetadata-plugin/blob/master/pg_metadata/dock.py#L55..L64

But here, we don't need extra attributes I think.
If you need `mapoql`, as it is in a single place, one time, just use the code python code provided https://github.com/3liz/QuickOSM/pull/293#discussion_r643585163